src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const resultObj = {};
+  const createdArray = sourceString.split(';');
+
+  createdArray.forEach((element) => {
+    const [key, value] = element.split(':');
+
+    if (key && value) {
+      resultObj[key.trim()] = value.trim();
+    }
```

Comment:
This `if` is useless because string with spaces will be like truth value 
![image](https://user-images.githubusercontent.com/122989585/218051553-f8fcd320-2f11-48b2-8ae7-b9f518466b22.png)
So you should trim it as well to check are there any symbols or not
```suggestion
    const trimmedKey = key.trim();
    const trimmedValue = value.trim();
    
    if (trimmedKey && trimmedValue ) {
      resultObj[trimmedKey] = trimmedValue ;
    }
```